cockatrice/src/abstractcounter.cpp

```diff
@@ -88,23 +88,25 @@ void AbstractCounter::setValue(int _value)
 
 void AbstractCounter::mousePressEvent(QGraphicsSceneMouseEvent *event)
 {
-    if (event->button() == Qt::LeftButton) {
-        Command_IncCounter cmd;
-        cmd.set_counter_id(id);
-        cmd.set_delta(1);
-        player->sendGameCommand(cmd);
-        event->accept();
-    } else if (event->button() == Qt::RightButton) {
-        Command_IncCounter cmd;
-        cmd.set_counter_id(id);
-        cmd.set_delta(-1);
-        player->sendGameCommand(cmd);
-        event->accept();
-    } else if (event->button() == Qt::MidButton) {
-        if (menu)
-            menu->exec(event->screenPos());
-        event->accept();
-    } else
+    if (isUnderMouse()) {
+        if (event->button() == Qt::LeftButton && isUnderMouse()) {
```

Comment:
Is there a reason to do an `isUnderMouse()` confirmation in the event case (line 92, 98, and 104) if the entire feature is under an initial if case to check for `isUnderMouse()` (line 91)?